mobile/src/reducers/people/people.reducer.ts

```diff
@@ -0,0 +1,48 @@
+import { Reducer } from 'redux';
+import { Map, Set } from 'immutable';
+import { combineEpics } from 'redux-observable';
+import { NavigationAction } from 'react-navigation';
+
+import { User } from '../user/user.model';
+import { Employee } from '../organization/employee.model';
+import { OrganizationActions, loadEmployeesForDepartment } from '../organization/organization.action';
+import { UserActions } from '../user/user.action';
+import { PeopleActions } from './people.action';
+import { EmployeeMap } from '../organization/employees.reducer';
+
+export interface PeopleState {
+    employees: EmployeeMap;
+    employeesDepartmentSubsetFilterCallback: any;
+    employeesRoomSubsetFilterCallback: any;
+    employeesSubsetFilterCallback: any;
+}
+
+const initState: PeopleState = {
+    employees: Map(),
+    employeesDepartmentSubsetFilterCallback: function(employee: Employee) {
```

Comment:
just a filter please, without Callback. It's just not in the js tradition of naming :) You may call it Predicate instead of FilterCallback, that's more acceptable in CS-world :)

Also, here is your typing (employee: Employee) => boolean;

Reply:
re: Callback - it was advice from Serj ;)

Reply:
not literally, as an example